src/backup/utils.ts

```diff
@@ -14,6 +14,7 @@ import { currentAccountSelector } from 'src/web3/selectors'
 const TAG = 'Backup/utils'
 
 const MNEMONIC_STORAGE_KEY = 'mnemonic'
+const ETHEREUM_DERIVATION_PATH = "m/44'/60'/0'"
```

Comment:
happy to move this to a config file but it's only used here, i see it declared as a variable like this in the celo monorepo too so didn't think too much further on where to place this